analyzers/tests/SonarAnalyzer.UnitTest/Rules/SymbolicExecution/ObjectsShouldNotBeDisposedMoreThanOnceTest.cs

```diff
@@ -18,30 +18,56 @@
  * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
  */
 
-using SonarAnalyzer.Rules.CSharp;
+using SonarAnalyzer.Common;
 using SonarAnalyzer.SymbolicExecution.Sonar.Analyzers;
+using ChecksCS = SonarAnalyzer.SymbolicExecution.Roslyn.RuleChecks.CSharp;
+using CS = SonarAnalyzer.Rules.CSharp;
 
 namespace SonarAnalyzer.UnitTest.Rules
 {
     [TestClass]
     public class ObjectsShouldNotBeDisposedMoreThanOnceTest
     {
-        private readonly VerifierBuilder sonar = new VerifierBuilder<SymbolicExecutionRunner>().WithBasePath(@"SymbolicExecution\Sonar")
+        private readonly VerifierBuilder sonar = new VerifierBuilder()
+            .AddAnalyzer(() => new CS.SymbolicExecutionRunner(AnalyzerConfiguration.AlwaysEnabledWithSonarCfg))
+            .WithBasePath(@"SymbolicExecution\Sonar")
             .WithOnlyDiagnostics(ObjectsShouldNotBeDisposedMoreThanOnce.S3966);
 
+        private readonly VerifierBuilder roslynCS = new VerifierBuilder()
+            // ToDo: .AddAnalyzer(() => new CS.SymbolicExecutionRunner(AnalyzerConfiguration.AlwaysEnabled))     // SE part
+            .WithBasePath(@"SymbolicExecution\Roslyn")
+            .WithOnlyDiagnostics(ChecksCS.ObjectsShouldNotBeDisposedMoreThanOnce.S3966);
+
         [DataTestMethod]
         [DataRow(ProjectType.Product)]
         [DataRow(ProjectType.Test)]
-        public void ObjectsShouldNotBeDisposedMoreThanOnce_CSharp8(ProjectType projectType) =>
+        public void ObjectsShouldNotBeDisposedMoreThanOnce_Sonar_CSharp8(ProjectType projectType) =>
             sonar.AddPaths("ObjectsShouldNotBeDisposedMoreThanOnce.cs")
                 .WithOptions(ParseOptionsHelper.FromCSharp8)
                 .AddReferences(TestHelper.ProjectTypeReference(projectType).Concat(MetadataReferenceFacade.NETStandard21))
                 .Verify();
 
+        [Ignore] // Fix after S3966 implementation
+        [DataTestMethod]
+        [DataRow(ProjectType.Product)]
+        [DataRow(ProjectType.Test)]
+        public void ObjectsShouldNotBeDisposedMoreThanOnce_Roslyn(ProjectType projectType) =>
```

Comment:
Becauase `_VB` is coming
```suggestion
        public void ObjectsShouldNotBeDisposedMoreThanOnce_Roslyn_CS(ProjectType projectType) =>
```